src/components/Display/CharTable.js

```diff
@@ -0,0 +1,91 @@
+import {
+  Table,
+  TableContainer,
+  TableHead,
+  TableBody,
+  TableRow,
+  TableCell,
+  Paper,
+  Avatar,
+  Box,
+} from '@mui/material';
+import React, { useState } from 'react';
+import CharModal from '../Modal/CharModal';
+
+const CharTable = ({ data }) => {
```

Comment:
Rename to something informative like `characterList`

Reply:
done